src/CommandingOfficers/AW3/AW3Commander.java

```diff
@@ -0,0 +1,54 @@
+package CommandingOfficers.AW3;
+
+import CommandingOfficers.Commander;
+import CommandingOfficers.CommanderInfo;
+import CommandingOfficers.TrilogyCommander;
+import CommandingOfficers.CommanderInfo.InfoPage;
+import Engine.GameScenario;
+import Units.UnitDelta;
+
+public abstract class AW3Commander extends TrilogyCommander
+{
+  private static final long serialVersionUID = 1L;
+  public static final InfoPage AW3_MECHANICS_BLURB = new InfoPage(
+      "Gain +10 attack and defense on activating a power.\n"
+    + "Power charge is not based on funds, but is a specific stat for each unit type.\n"
```

Comment:
So like, power could be awarded based on HP for infantry, move distance for recons, armor rating for bopters, etc?

Reply:
I mean, kind of.
Recons have the highest move on a ground unit, and give the most charge-per-funds of anything but Infantry.